### PR TITLE
can-1397

### PR DESCRIPTION
**Pull request by shahab-ramzan**
Object button typo fixed

### CODE REVIEW

File: src/components/ComponentPages/HistoryContainer/Collapse/index.tsx [hunk 1]
```diff
@@ -228,7 +228,7 @@ function HistoryCollapse({
                       Object
                     </Button>
                     <Modal
-                      title="Why can't i object?"
+                      title="Why can't I object?"
```
Comment (akuks) [inline src/components/ComponentPages/HistoryContainer/Collapse/index.tsx]: I would suggest, move it to common message file

File: src/components/ComponentPages/HistoryContainer/Collapse/index.tsx [hunk 2]
```diff
@@ -245,11 +245,11 @@ function HistoryCollapse({
                           Close
                         </Button>,
                       ]}
+                      onCancel={() => setModal1Open(false)}
                     >
                       <p>
-                        To object to this change, you should be a direct
-                        supporter of the topic/camp and should have supported it
-                        before the change was submitted.
+                        Only direct supporters at the time this change was
```
Comment (akuks) [inline src/components/ComponentPages/HistoryContainer/Collapse/index.tsx]: I would suggest, move it to common message file

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/HistoryContainer/Collapse/index.tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/HistoryContainer/Collapse/index.tsx]: Done

File: src/components/ComponentPages/HistoryContainer/Collapse/index.tsx (hunk 2 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/HistoryContainer/Collapse/index.tsx]: Done

**Review by akuks — APPROVED** [review]

**Pull request merged**